configTools/src/test/scala/com/gu/janus/testutils/HaveMatchers.scala

```diff
@@ -0,0 +1,18 @@
+package com.gu.janus.testutils
+
+import org.scalatest.matchers.HavePropertyMatcher
+import org.scalatest.matchers.should.Matchers
+
+
+trait HaveMatchers extends Matchers {
```

Comment:
This helper restores the super-useful `have` matcher for Scala 2.13. Symbols are deprecated so you can no longer write this the way scalatest expects.

Old way:
```scala
foo should have(
  'fieldName (value),
  'fieldName2 (value2),
)
```

New way
```scala
foo should have(
  "fieldName" as value,
  "fieldName2" as value2,
)
```